react/__tests__/Backdrop.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { render, fireEvent } from '@vtex/test-tools/react'
+
+// eslint-disable-next-line jest/no-mocks-import
+import { findCSSHandles } from '../__mocks__/testUtils'
+import Backdrop, { CSS_HANDLES } from '../components/Backdrop'
+
+describe('<Backdrop />', () => {
+  it('should render', () => {
+    const { baseElement } = render(<Backdrop open />)
+
+    expect(baseElement).toBeTruthy()
+  })
+
+  it('should find all declared handles', () => {
+    const { container } = render(<Backdrop open />)
+
+    const foundHandles = findCSSHandles(container, CSS_HANDLES)
+    expect(foundHandles).toEqual(CSS_HANDLES)
+  })
+
+  it('should have visibility hidden and opacity 0 if open === false', () => {
+    const { getByTestId } = render(<Backdrop open={false} />)
+
+    const container = getByTestId('modal-backdrop-container')
+
+    expect(container.style.getPropertyValue('opacity')).toBe('0')
+    expect(container.style.getPropertyValue('visibility')).toBe('hidden')
+  })
+
+  it('should call onClick when backdrop is clicked', () => {
+    const spy = jest.fn()
```

Comment:
Why is this called "spy"? 👀

Reply:
Because it is one. I just have to know if this function is called https://jestjs.io/docs/en/mock-function-api